peachjam/views/courts.py

```diff
@@ -1,69 +1,36 @@
 from itertools import groupby
 
+from django.http import Http404
 from django.shortcuts import get_object_or_404
-from django.urls import reverse
 from django.utils.dates import MONTHS
 
 from peachjam.helpers import chunks, lowercase_alphabet
-from peachjam.models import Court, CourtRegistry, Judgment
+from peachjam.models import Court, CourtClass, CourtRegistry, Judgment
 from peachjam.views.generic_views import FilteredDocumentListView
 
 
-class CourtDetailView(FilteredDocumentListView):
-    """List View class for filtering a court's judgments.
-
-    This view may be restricted to a specific year.
-    """
+class BaseJudgmentFilterView(FilteredDocumentListView):
```

Comment:
minor: maybe `FilteredJudgmentListView` is a more consistent name.

Reply:
changed